src/api/app/assets/javascripts/webui/request.js

```diff
@@ -107,3 +107,23 @@ function requestAddAutocomplete(autocompleteElement) { // jshint ignore:line
     $('.hideable input:visible').removeAttr('disabled');
   });
 }
+
+$(document).ready(function(){
+  $('a[data-toggle="tab"]').on('shown.bs.tab', function () {
```

Comment:
With this code, we're loading diffs whenever a tab is clicked. This is great.

There's one issue though. Let's say that we have a request with 2 tabs: _Tab 1_ and _Tab 2_. So when the page is loaded, _Tab 1_ will also be loaded. I then click on _Tab 2_ and its diffs are asynchronously loaded. I go back to _Tab 1_ and its diffs are loaded **again**. So whenever I click on a tab for which its diffs were previously loaded, they are loaded again since this code is sending another AJAX request. This should be avoided.

Reply:
This suggestion is already tackled. We have added new commits to make it easier to understand the steps we followed. We'll like to squash them before it gets merged.

Can you please have a look, @dmarcoux ?

Reply:
Done. It's updated. We don't send a new AJAX now.